src/assignment-logger.ts

```diff
@@ -33,6 +35,16 @@ export interface IAssignmentEvent {
    */
   timestamp: string;
 
+  /**
+   * An Eppo holdout key
+   */
+  holdout: string | null;
+
+  /**
+   * The Eppo holdout variation for the assigned variation
+   */
+  holdoutVariation: HoldoutVariationType;
```

Comment:
Should this also be optionally null?
```ts
holdoutVariation: HoldoutVariationType | null;
```

Reply:
`HoldoutVariationType` is `'status_quo' | 'all_shipped_variants' | null`

I could rename this to `HoldoutVariationNullableType` so it's more obvious